internal/metrics/metric.go

```diff
@@ -0,0 +1,44 @@
+package metrics
+
+import "github.com/mrkovshik/yametrics/internal/storage/server"
+
+type Imetric interface { //TODO: Подумать, нужен ли тут интерфейс вообще
+	Update(server.IStorage) error
+}
+
+const (
+	MetricTypeGauge   = "gauge"
+	MetricTypeCounter = "counter"
+)
+
+var MetricNamesMap = map[string]struct{}{
```

Comment:
CHANGE REQUEST: please don't use global variables

Reply:
I moved this map inside of the SendMetric method, but I'm not quite sure if this is the best solution. I feel like it should be available all over the project (however it forks just fine now).